job-dsls/src/main/groovy/org/kie/jenkins/jobdsl/Constants.groovy

```diff
@@ -23,6 +23,10 @@ class Constants {
     static final String BRANCH = "master"
     static final String GITHUB_ORG_UNIT = "kiegroup"
     static final String PULL_REQUEST_FOLDER = "pullrequest"
+    static final String FDB_FOLDER = "fullDownstream"
+    static final String CDB_FOLDER = "compileDownstream"
+    static final String UPSTREAM_FOLDER = "upstream"
+    static final String DOWNSTREAM_PRODUCT_FOLDER = "downstream.product"
```

Comment:
`downstream-production` would probably be a more appropriated name

Reply:
i think production is not correct term here, production is the status or environment 
Here we do really project/product distinction